src/applications/personalization/profile/sass/profile-info-table.scss

```diff
@@ -1,6 +1,6 @@
 $corner-size: 4px;
 
-.profile-info-table {
+.profile-info-table, .profile-info-card {
```

Comment:
For this scss file in general, if we can determine that the border radius isn't needed on this table, then we _could_ get rid of this whole file (the border is the only custom part of this really). 

I'm confident that the VA design system classes can be composed to accomplish the same goals without custom styling.